### PR TITLE
feat(enum-rule): implement rule key and value should be identical

### PR DESCRIPTION
**Pull request by thaisrodez**

### CODE REVIEW

File: src/fixture/file.ts
Comment (yannicktian) [inline src/fixture/file.ts:1]: Is this empty file useful?

File: src/fixture/react.tsx
Comment (yannicktian) [inline src/fixture/react.tsx:1]: Same, do you need this file?

File: src/fixture/tsconfig.json
Comment (yannicktian) [inline src/fixture/tsconfig.json:1]: Why do you need this?

File: yarn-error.log
Comment (yannicktian) [inline yarn-error.log:1]: You don't need to commit this file

File: src/fixture/file.ts
Comment (thaisrodez) [reply src/fixture/file.ts:1]: yes, it is usefull to the test, see docs here : https://typescript-eslint.io/packages/rule-tester/#type-aware-testing

**Pull request merged**